File: LeetCode/Medium/1291-sequential-digits/1291-sequential-digits.java
class Solution {
    public List<Integer> sequentialDigits(int low, int high) {
        List<Integer> ret = new ArrayList();
        
        for (int i = 1; i < 10; ++i){
            int j = i + 1;
            int temp = i;
            
            while (j < 10 && temp < high){
                temp = (temp * 10) + j;
                ++j;
                
                if (temp >= low && temp <= high) ret.add(temp);
            }
        }
        
        Collections.sort(ret);
        
        return ret;
    }
}

File: LeetCode/Medium/1291-sequential-digits/1291-sequential-digits.go
func sequentialDigits(low int, high int) []int {
    ret := make([]int, 0);
        
    for i := 1; i < 10; i++ {
        j := i + 1
        temp := i

        for j < 10 && temp < high {
            temp = (temp * 10) + j
            j++

            if temp >= low && temp <= high { ret = append(ret, temp) }
        }
    }

    sort.Ints(ret)

    return ret
}